jax/_src/scipy/linalg.py

```diff
@@ -1137,6 +1137,21 @@ def expm(A: ArrayLike, *, upper_triangular: bool = False, max_squarings: int = 1
 
   See Also:
     :func:`jax.scipy.linalg.expm_frechet`
+
+  Example:
```

Comment:
I think this is a poor example. Examples should show give insight into why a function is useful, not simply show that you can call a function and get numbers out without any broader context.

What if we demonstrated one of the properties of the matrix exponential; e.g. that `expm(A + B)` is equivalent to `expm(A) @ expm(B)` for commuting matrices `A` and `B`?

Reply:
Thanks for the suggestion. Updated accordingly.